src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -322,6 +322,28 @@ def select_frame(self, locator):
 
         Key attributes for frames are `id` and `name.` See `introduction` for
         details about locating elements.
+        
+        See `Unselect Frame` to cancel the frame selection and return to the Main frame.
+        
+        Please note that the frame search always start from the document root or main frame.
+        
+        Examples:
+        | # Two frame elements with equal names | | |
+        | Select Frame   | xpath=`/``/`frame[@name='left']`/`iframe[@name='left'] | # Selects the 'left' iframe |
+        | Click Link     | foo                                                    | # Clicks link 'foo' in 'left' iframe |
+        | Unselect Frame |                                                        | # Returns to main frame |
+        | Select Frame   | left                                                   | # Selects the 'left' frame |
+  
+        | # Multiple frames with unique names or id | | |
```

Comment:
I do not think that having this example brings value compared to the first example. Please remove it.

Reply:
This example shows (was meant to show) that having three frames on the same level, one has to unselect them to return to search point. Did not want to complicate more the example by showing the frame's structure. But maybe we should?

Reply:
But that is already shown in the first example. Select frame, do something and unselect the frame. Having second similar example doesn't add value.